README.md

```diff
@@ -421,6 +459,9 @@ with a PCI vendor ID of `15b3` _AND_ PCI device ID of `1014` _or_ `1017`.
 - A node would contain the label: `feature.node.kubernetes.io/custom-my.accumulated.feature=true`
 if `some_kmod1` _AND_ `some_kmod2` kernel modules are loaded __OR__ the node contains a PCI device
 with a PCI vendor ID of `15b3` _AND_ PCI device ID of `1014` _OR_ `1017`.
+- A node would contain the label: `feature.node.kubernetes.io/custom-my.kernel.featureneedscpu=true`
+if `KVM_INTEL` kernel config is enabled __AND__ the node CPU supports `VMX` virtual machine extensions
+- A node would contain the label: `feature.node.kubernetes.io/custom-my.kernel.modulecompiler=true` if `kmod1` kernel module is loaded __AND__ it's built with `GCC_VERSION=100101`.
```

Comment:
> `kmod1` kernel module is loaded __AND__ it's built with `GCC_VERSION=100101`.

This may not always be the case right ? e.g `kmod1` is an out of tree kernel module which may have been compiled with a different compiler than the kernel itself.

Reply:
changed to "if the in-tree `kmod1` kernel module is loaded"